Parser/pegen/pegen.c

```diff
@@ -337,9 +352,6 @@ tokenizer_error(Parser *p)
             errtype = PyExc_IndentationError;
             msg = "too many levels of indentation";
             break;
-        case E_DECODE:
-            raise_decode_error(p);
-            return -1;
```

Comment:
Alright, I didn't understand this code path. I now understand better raise_decode_error() with the assertion, thanks ;-)